src/averageAges.js

```diff
@@ -38,6 +54,14 @@ function calculateMenAverageAge(people, century) {
  */
 function calculateWomenAverageAge(people, withChildren) {
   // write code here
+  const filteredWomen = withChildren
+    ? people.filter(person => people.find(child => isMother(child, person)))
+    : people.filter(person => person.sex === 'f');
```

Comment:
since you have a helper function `isMan` - you could use the opposite of it:
```suggestion
    : people.filter(!isMan(person));
```